src/js/demo/components/App.jsx

```diff
@@ -6,7 +6,7 @@ import BugReport from "./BugReport";
 import Crash from "./Crash";
 import Configuration from "./Configuration";
 import Unicode from "../utils/unicode";
-import linterModule from "../../eslint";
+import linterModule from "../node_modules/eslint/lib/linter/linter";
```

Comment:
I thought it was better to be explicit here because we don't ever want to accidentally install the top level eslint package we're using for linting.